compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -284,6 +287,93 @@ void addPackBasedPassPipeline(OpPassManager &pm, TilingConfig &tilingConfig) {
   addAMDAIEBufferizePasses(modulePassManager);
 }
 
+void addPadPackBasedPassPipeline(OpPassManager &pm,
+                                 TilingConfig &tilingConfig) {
+  auto &modulePassManager = pm.nest<ModuleOp>();
+  modulePassManager.addNestedPass<func::FuncOp>(createAMDAIECleanupPass());
+  pm.addPass(createCanonicalizerPass());
+  pm.addPass(createCSEPass());
+
+  AMDAIETileOptions tileOptions;
```

Comment:
Same here, Lets not reuse the same options structs across different pass invocations.